src/app/(withAuth)/works/[workId]/edit/page.tsx

```diff
@@ -20,8 +22,14 @@ export default async function Edit({
 }: {
   params: { workId: string }
 }) {
+  const session = await getServerSession(authOptions)
+  if (!session) {
+    redirect('/')
```

Comment:
セッションがない時に`redirect`する実装方法はとても自然なのですが、他のログイン必須ページでは現状`notFound`で終わらせている感じになっているので, 今は`notFound`で終わらせて大丈夫です！

今後, セッション必須の部分でセッションがない場合に, `notFound`にするか`redirect`にするか検討する必要があると思っています。方法が決まり次第, 別のプルリクエストでリファクタリングする予定なので, 本PRでは保留で問題ないです。

```suggestion
    notFound()
```